maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -3474,6 +3552,14 @@ public void setAdditionalClasspathElements(String[] additionalClasspathElements)
         this.additionalClasspathElements = additionalClasspathElements;
     }
 
+    public Dependency[] getAdditionalClasspathDependencies() {
+        return additionalClasspathDependencies;
+    }
+
+    public void setAdditionalClasspathDependencies(Dependency[] additionalClasspathDependencies) {
+        this.additionalClasspathDependencies = additionalClasspathDependencies;
+    }
```

Comment:
we don't use this values outside class - so getter and setter are not needed.

Reply:
Right, this was again to be in line with other parameters, but agree we don't need them.